pkg/aggregator/aggregator.go

```diff
@@ -246,6 +246,16 @@ func (agg *BufferedAggregator) addServiceCheck(sc metrics.ServiceCheck) {
 	agg.serviceChecks = append(agg.serviceChecks, &sc)
 }
 
+func (agg *BufferedAggregator) addSerie(serie metrics.Serie) metrics.Series {
```

Comment:
don't think it makes sense to have an `addSerie` function here given how the series are aggregated